packages/muelu/test/vardofpernode/CMakeLists.txt

```diff
@@ -6,56 +6,87 @@ TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../../src/Smoothers)
 TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../../gallery)
 TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../../adapters/belos)
 
-IF (${PACKAGE_NAME}_ENABLE_Belos)
+IF(${PACKAGE_NAME}_ENABLE_Belos)
 
-TRIBITS_ADD_EXECUTABLE(
+  TRIBITS_ADD_EXECUTABLE(
     VarDofDriver
     SOURCES VarDofDriver.cpp
     COMM serial mpi
   )
 
 ENDIF()
 
-TRIBITS_COPY_FILES_TO_BINARY_DIR(VarDofDriver_cp
-  SOURCE_FILES xxx.mm yyy.mm theMatrix.m rhs.mm 
-  driver.xml driver_EminDistLapl.xml 
-  1ProcLinear0 1proc0 1dofPresent0 
-  2ProcLinear0  2ProcLinear1  2dofPresent0  2dofPresent1  2proc0  2proc1 
-  7ProcLinear0 7ProcLinear1 7ProcLinear2 7ProcLinear3 7ProcLinear4 7ProcLinear5 7ProcLinear6 7dofPresent0 7dofPresent1 7dofPresent2 7dofPresent3 7dofPresent4 7dofPresent5 7dofPresent6 7proc0 7proc1 7proc2 7proc3 7proc4 7proc5 7proc6
+TRIBITS_COPY_FILES_TO_BINARY_DIR(
+  VarDofDriver_cp
+  SOURCE_FILES
+    xxx.mm
+    yyy.mm
+    theMatrix.m
+    rhs.mm
+    driver.xml
+    driver_EminDistLapl.xml
+    1ProcLinear0
+    1proc0
+    1dofPresent0
+    2ProcLinear0
+    2ProcLinear1
+    2dofPresent0
+    2dofPresent1
+    2proc0
+    2proc1
+    7ProcLinear0
+    7ProcLinear1
+    7ProcLinear2
+    7ProcLinear3
+    7ProcLinear4
+    7ProcLinear5
+    7ProcLinear6
+    7dofPresent0
+    7dofPresent1
+    7dofPresent2
+    7dofPresent3
+    7dofPresent4
+    7dofPresent5
+    7dofPresent6
+    7proc0
+    7proc1
+    7proc2
+    7proc3
+    7proc4
+    7proc5
+    7proc6
```

Comment:
I like this.

Reply:
@cgcgcg Is clang formatting imposing/enforcing an alpha-numeric ordering?  That would be nice to have.

Reply:
Good suggestion! Done.